docs/content/preview/yugabyte-cloud/cloud-clusters/cloud-maintenance.md

```diff
@@ -62,14 +62,20 @@ To set the maintenance exclusion period for a cluster:
 
 ## What to expect during maintenance
 
-Yugabyte performs rolling maintenance and upgrades on [fault tolerant](../../cloud-basics/create-clusters-overview/#fault-tolerance) clusters with zero downtime. However, the cluster is still subject to the following:
+The impact of maintenance on a cluster depends on its topology and [fault tolerance](../../cloud-basics/create-clusters-overview/#fault-tolerance).
 
-- Dropped connections - Connections to the stopped node are dropped. Verify your connection pool, driver, and application to ensure they handle dropped connections correctly. Any failures need to be retried.
+### Fault tolerance none
```

Comment:
Since we are referring to the other section as "Fault tolerant clusters" this may sound better worded similarly such as "No fault tolerance clusters". Or we can change the approach of both and do (1) Clusters with no fault tolerance (2) Clusters with fault tolerance